src/io/cpal.rs

```diff
@@ -366,7 +382,7 @@ impl AudioBackendManager for CpalBackend {
     }
 
     fn output_latency(&self) -> f64 {
-        self.output_latency.load(Ordering::SeqCst)
+        self.output_latency.load(Ordering::Relaxed)
```

Comment:
I choose the relaxed ordering because I don't think this is really important, let me know if you see any issue with that

Reply:
Yes, this is fine